libsolidity/DeclarationContainer.h

```diff
@@ -42,8 +42,8 @@ class DeclarationContainer
 	explicit DeclarationContainer(Declaration const* _enclosingDeclaration = nullptr,
 								  DeclarationContainer const* _enclosingContainer = nullptr):
 		m_enclosingDeclaration(_enclosingDeclaration), m_enclosingContainer(_enclosingContainer) {}
-	/// Registers the declaration in the scope unless its name is already declared.
-	/// @returns true iff it was not yet declared.
+	/// Registers the declaration in the scope unless its name is already declared or the name is empty.
+	/// @returns true if the declaration was registered.
```

Comment:
If the name is empty, nothing is registered, so it should rather read "returns false if the name was already declared.".